canary/mqtt5/canary.ts

```diff
@@ -101,119 +150,126 @@ async function doUnsubscribe(context : CanaryContext) {
     }
 }
 
-async function doPublish(context : CanaryContext, qos: mqtt5_packet.QoS) {
+async function doPublish(context: CanaryContext, qos: mqtt5.QoS) {
     try {
         context.mqttStats.publishesAttempted++;
 
-        await context.client.publish({
+        // Generate random binary payload data
+        let payload = Buffer.alloc(10000);
+        for(let i = 0; i < 10000; i++)
+        {
+            payload[i] = Math.floor(Math.random() * 128);
```

Comment:
A question: does initializing buffer with random bytes affect publishing in any way?

Reply:
I thought about it, and it is probably not necessary for canary. Using a random bytes for payload is more for "functional test". It can make sure we handle the string encoding correctly. However, it is not a required for canary. 

The payload size still matters, as a large payload size would make it easier to observe memory issue related to payload, but the payload data is not necessary a focus. 

I will revert it.

Reply:
I see, thanks for the explanation!